natgeo/mortar
# [SYTO-1163] Mortar home page modifications

https://jira.natgeo.com/browse/SYTO-1163

To test: 

- Check out the Brand new mortar homepage
- Make sure the AC is met
